inst/qml/Power.qml

```diff
@@ -56,124 +60,164 @@ Form
         Text { text: qsTr("I want to calculate the ...") }
         DropDown
         {
-          name: "calc"
+          name: "calculation"
           id:   calc
           indexDefaultValue: 0
           label: qsTr("")
           values: [
-            { label: "Sample Size N", value: "n"},
+            { label: "Sample Size N", value: "sampleSize"},
             { label: "Power",  value: "power" },
-            { label: "Effect size",  value: "es"}
+            { label: "Effect size",  value: "effectSize"}
           ]
         }
 
         Text {
-          text: qsTr("First proportion")
-          enabled: test.currentIndex == 5
+          text: qsTr("Direction of the effect:")
+          visible: (test.currentIndex == 4 || test.currentIndex == 5 || test.currentIndex == 6 || test.currentIndex == 7) && calc.currentIndex == 2 && alt.value == "twoSided"
+          enabled: calc.currentIndex == 2
+        }
+				DropDown 
+        {
+          id: direction
+          name: "effectDirection"
+          label: qsTr("")
```

Comment:
Hi, we shoud not mark a empty string as to be translated:-) and [here](https://github.com/jasp-stats/jaspModuleTemplate/pull/8) we will have a test to check this.